tests/vmi_multus_test.go

```diff
@@ -942,6 +942,64 @@ var _ = Describe("[Serial]SRIOV", func() {
 				})
 			})
 		})
+
+		Context("migration", func() {
+
+			BeforeEach(func() {
+				nodes := tests.GetAllSchedulableNodes(virtClient)
+				Expect(nodes.Items).ToNot(BeEmpty(), "There should be at least one compute node")
+
+				if len(nodes.Items) < 2 {
```

Comment:
As we discussed we should improve this 
We could have a function that will check that there are at least two nodes with SRIOV devices resource
 similar to [checkSriovEnabled](https://github.com/kubevirt/kubevirt/pull/4741/commits/91b74325a9f1fafeeaa746a53d6e0b586393b5c3#diff-ac233e621be1a1855fe741915d860d91484c7f061fc42beb213f4f9e0a756f86R1413-R1428)